state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -703,22 +704,27 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			RequestFailureCallbacks::<T, I>::insert(broadcast_id, callback);
 		}
 
-		(broadcast_id, Self::threshold_sign(api_call, broadcast_id, true))
+		(broadcast_id, Self::threshold_sign(api_call, broadcast_id, true, false))
 	}
 
 	/// Signs a API call, use `Call::on_signature_ready` as the callback, and returns the signature
 	/// request ID.
 	fn threshold_sign(
-		api_call: <T as Config<I>>::ApiCall,
+		mut api_call: <T as Config<I>>::ApiCall,
 		broadcast_id: BroadcastId,
 		should_broadcast: bool,
+		refresh_replay_protection: bool,
```

Comment:
This value is always false except when used from the governance call.

I would suggest we simple move the conditional into the body the extrinsic instead, then we don't need to thread the boolean value through all the other functions where it isn't used.